tox.ini

```diff
@@ -26,19 +24,21 @@ extras =
 commands =
   ruff format --check qiskit_addon_sqd/ docs/ test/
   ruff check qiskit_addon_sqd/ docs/ test/
+  ruff check --preview --select CPY001 --exclude "*.ipynb" qiskit_addon_sqd/ test/
   nbqa ruff docs/
+  pydocstyle qiskit_addon_sqd/
   mypy qiskit_addon_sqd/
-  #reno lint
   pylint -rn qiskit_addon_sqd/ test/
   nbqa pylint -rn docs/
-  pydocstyle qiskit_addon_sqd/
+  typos
+  reno lint
 
-[testenv:{,py-,py3-,py39-,py310-,py311-,py312-}notebook]
+[testenv:{,py-,py3-,py310-,py311-,py312-}notebook]
 extras =
   nbtest
   notebook-dependencies
 commands =
-  pytest --nbmake --nbmake-timeout=3000 {posargs} docs/tutorials/
+  pytest --nbmake --nbmake-timeout=3000 {posargs} docs/
```

Comment:
You actually did not test the how-tos because of this.

Reply:
Looks like this breaks CI now. Are some guides running very long?
Over in `qiskit-addon-mpf`, I added `skip-execution` tags to cells which run too long for CI. For example here: https://github.com/Qiskit/qiskit-addon-mpf/blob/0085bcf5e24d829ca54080a9b8478573b652417b/docs/explanations/mpf_stability.ipynb#L158

Reply:
Yes there is a benchmarking guide that runs quite a while. Also, I doubt dice solver how-to will run since it is not in the notebook environments and needs to be built more manually (can possibly get it tested in CI soon)